.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,24 @@
+name: Continous Integration
+on:
+  pull_request:
+    branches:
+      - master

```

Comment:
include `linux` here.

Reply:
I wonder if we need to merge this in order to enable it.

Reply:
That's what I'm guessing. And I fear it needs to merge into master. I recalled we talked about merging `linux` in to `master`. I think this would simplify our CI setup and help us move from one functional point to another. But if this wouldn't work with he current CI / shipping process we can relegate to the `linux` branch. Given there's not a GH actions - is there an Azure pipeline check that runs Windows CI?

Reply:
There is an Azure Pipeline, but it's probably only checked on PRs to `master`.